packages/react/index.ts

```diff
@@ -217,7 +220,7 @@ function checkAndReport(node: Node, timeout: number): Promise<void> {
               const key: string = node.target.toString() + result.id;
               const retVal = !cache[key];
               cache[key] = key;
-              return retVal;
+              return disableCache || retVal;
```

Comment:
This isn't really preventing the cache from being used, as much as it's just overriding it. Probably better to just prevent it from even calling `violations.filter`.

Reply:
Yeah so I sort of did this intentionally. I was thinking that some invocations of the library might want to leverage cache from previous runs?

Example, a hot reload would ignore cache and do a full accessibility test to verify that an issue was fixed properly but secondary data load from a page might only report on new errors. Does that make sense?

Reply:
Ahhh, I see what you mean. I like it.